modules/openthread/Kconfig

```diff
@@ -17,6 +17,21 @@ config OPENTHREAD
 
 if OPENTHREAD
 
+config OPENTHREAD_SYS_INIT
+	bool "Initialize OpenThread stack during system initialization"
+	default y if !NET_L2_OPENTHREAD
```

Comment:
Should we rather disable this option if OpenThread L2 is enabled? Otherwise it'd be theoretically possible to initialize OT stack twice on boot, which makes little sense I think? Something like:
```
depends on !NET_L2_OPENTHREAD
default y
```

Reply:
Ok, it makes sense.